recipes/mongodb_org_repo.rb

```diff
@@ -38,7 +38,7 @@
 when 'rhel', 'fedora'
   yum_repository 'mongodb' do
     description 'mongodb RPM Repository'
-    baseurl "http://downloads-distro.mongodb.org/repo/redhat/os/#{node['kernel']['machine']  =~ /x86_64/ ? 'x86_64' : 'i686'}"
+    baseurl "#{node['mongodb']['repo']}/#{node['kernel']['machine']  =~ /x86_64/ ? 'x86_64' : 'i686'}"
```

Comment:
Yes I think it will be cleaner too. If you want you can update the branch directly https://github.com/chef-brigade/mongodb-cookbook/tree/pr/387 and merge.